api/src/main/kotlin/io/quarkus/code/misc/QuarkusExtensionUtils.kt

```diff
@@ -68,7 +68,11 @@ object QuarkusExtensionUtils {
         val data = extension.syntheticMetadata
         for (entry in data.entries) {
             if (TAG_KEYS.any { Regex(it).matches(entry.key) } && !entry.value.isEmpty()) {
-                tags.add(entry.key + ":" + entry.value.first())
+                if ("supported-by".equals(entry.key)) {
+                    tags.add(entry.value.first() + ":" + data.get(entry.value.first() + "-support")?.first())
```

Comment:
```suggestion
                   final String key = entry.value.first() + "-support";
                     tags.add(key + ":" + data.get(key)?.first())
```

You need to do this to keep compatibility 👆

Reply:
Will do.